index.html

```diff
@@ -5219,12 +5333,11 @@ <h2>Base Context</h2>
         "id": "@id",
         "type": "@type",
 
-        "cred": "https://www.w3.org/2018/credentials#",
```

Comment:
The content integrity protection for the base context is going to be destroyed w/ this search/replace. A decision will have to be made if we want to keep supporting the old v1 context or we want new v2 contexts everywhere. I suggest we upgrade everything and don't mention the v1 context. I expect the v2 examples context will need to change as well (because of `name`, and `description`).

Reply:
Could you elaborate on this a little more and what you mean by the content integrity protection?